[ios] Add instructions to run Organic Maps on a real device (#7181)
organicmaps/organicmaps: docs/INSTALL.md

- Click "+" and choose "Apple Development".
- You may also need to register your Mac in your Apple Developer account.

If you want to run Organic Maps on a real device, you need a CarPlay entitlement. You can either request the entitlement for your own
bundle identifier from Apple (see [here](https://developer.apple.com/documentation/carplay/requesting_carplay_entitlements)) or you join

> the Organic Maps Apple Developer team. Please contact us to request access to our team.

Reviewer: I remember that it was possible to run OM without CarPlay support by removing entitlement. Did you manage to do that?

Author: Ah no I'll try that later and get back to you. Thanks!

Author: @biodranik You're right. I remove the entitlement and now I can install Organic Maps on my iPhone if I select a different team. I've updated `INSTALL.md` with the steps to remove the entitlement.

- Click "+" and choose "Apple Development".
- You may also need to register your Mac in your Apple Developer account.

If you want to run Organic Maps on a real device, you have to be part of the Organic Maps development team. You can't run the app under

> your own team because you don't have the CarPlay entitlement. Please contact us to request access to the developemnt team.

Reviewer: Well, you can setup your own CarPlay entitlement...

Author: I contacted Apple and tried to get an entitlement. They've never responded to my request and after one month I don't think there will be any response. I just think future developer should be aware of that to safe them some time.

Reviewer: At least its worth mentioning in the docs.

Author: I added the entitlement part. Let me know if I should changed the wording.

- Open `xcode/omim.xcworkspace` in Xcode.
- Click on "Maps" project.
- Open "Signing & Capabilities" tab.
- Choose a unique bundle identifier (not app.organicmaps.debug) and your team.
- Select "Automatically manage signing".

If you want to run Organic Maps on a real device, you have to remove the CarPlay entitlement. Open `iphone/Maps/OMaps-Debug.entitlements`

> and remove the `com.apple.developer.carplay-maps` entry. Now you can sign your app again in the "Signing & Capabilities" tab. You can 
> only test a CarPlay feature on a real device if you're part of the Organic Maps Apple Developer team. Please contact us to request access
> to the team.

Reviewer: ```suggestion
and remove the `com.apple.developer.carplay-maps` entry. Now you can sign your app again in the "Signing & Capabilities" tab. Testing CarPlay
on a real device requires [requesting entitlements from Apple](https://developer.apple.com/documentation/carplay/requesting_carplay_entitlements).
```
